_layouts/prototype.html

```diff
@@ -3,39 +3,66 @@
 ---
 
 <div class="prototype">
-  <h2>{{ page.title }}</h2>
-  <p>{{ page.description }}</p>
-  
-  {% if page.prototype_url %}
-    <p><a href="{{ page.prototype_url }}">Try it now!</a></p>
-  {% endif %}
+  <div class="container">
+    <div class="col1">
+      <div class="row">
+        <p>
+          {{ page.category }} 
```

Comment:
The category should have some more space to the right of it before the license button